values.yaml

```diff
@@ -22,7 +22,7 @@ config:
 
 image:
   repository: "tryretool/backend"
-  tag: "latest"
+  tag: "2.66.65"
```

Comment:
As I was writing the README update, I thought, could we set this to `X.XX.XX`? This would make the user deliberately set a version.

It would help specify what new or existing users what to put here, and prevent the problem we were talking about (i.e. existing users merging this change, not noticing the version change, and unwittingly getting a different version than they expected).

What do you think @apurvas?

Reply:
Discussed offline. This sounds good.